package.json

```diff
@@ -97,7 +97,7 @@
     "@linkurious/ogma": "5.1.1",
     "@linkurious/ogma-annotations": "1.1.16",
     "@linkurious/ogma-export-stitch": "0.0.32",
-    "@linkurious/ogma-linkurious-parser": "4.1.4",
+    "@linkurious/ogma-linkurious-parser": "4.1.5-master-stable.2",
```

Comment:
will it be merged like this?

Reply:
yes, as we still haven't done the release, that is the dependency on the master release branch.

Reply:
This has been procedured in the past I beleive.
Either it does not need 4.1.5, and thus you can leave 4.1.4
Or it requires a release during the release process, and this is permitted by the human validation of the server PR.